test/helper.js

```diff
@@ -17,24 +20,30 @@ export async function resetGitRepository() {
 }
 
 async function initRepo() {
-  await git.init();
-  git.addConfig('user.name', config.get('history.author').name)
-     .addConfig('user.email', config.get('history.author').email);
+  for (const repoPath of [VERSIONS_PATH, SNAPSHOTS_PATH]) {
+    const git = new Git(repoPath);
+    await git.init();
+    git.addConfig('user.name', config.get('history.author').name)
+       .addConfig('user.email', config.get('history.author').email);
+  }
 }
 
 async function eraseRepo() {
-  const DATA_PATH = path.resolve(__dirname, '../', config.get('history.dataPath'));
-  const files = await fs.readdir(DATA_PATH, { withFileTypes: true });
-
-  const promises = files.map(file => {
-    const filePath = path.join(DATA_PATH, file.name);
-
-    if (file.isDirectory()) {
-      return fs.rmdir(filePath, { recursive: true });
-    } else if (file.name !== 'README.md') {
-      return fs.unlink(filePath);
-    }
-  });
+  const promises = [];
+
+  for (const repoPath of [VERSIONS_PATH, SNAPSHOTS_PATH]) {
+    const files = await fs.readdir(repoPath, { withFileTypes: true });
+
+    promises.push(...files.map(file => {
```

Comment:
```suggestion
    promises.concat(files.map(file => {
```

Reply:
Not exactly :). It should be:
```
promises = promises.concat(files.map(file => {
```
so, instead of 
```
promises.push(...files.map(file => {
```
I'm not sure which one is better…? Do you have any preference?

Reply:
Ugh… Thanks for the correction and sorry for the noise.

Both look as complex now. I probably have a small preference for `.push` as it is clearer that it is an in-place operation extending the array rather than a full change of its content.